.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "Secrets Store CSI Driver Provider for Azure",
+	"dockerFile": "Dockerfile",
+	"mounts": [
+		"source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
+    "source=${env:HOME}${env:USERPROFILE}/.azure,target=/home/vscode/.azure,type=bind"

```

Comment:
indent
Some shells will resolve both `$HOME` and `$USERPROFILE` - check this.

Reply:
```
[1776 ms] C:\Program Files\Docker\Docker\resources\bin\docker.exe: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /.azure.
See 'C:\Program Files\Docker\Docker\resources\bin\docker.exe run --help'.

[1777 ms] Failed: Starting the development container
[1794 ms] Command failed: C:\Program Files\Docker\Docker\resources\bin\docker.exe run -a STDOUT -a STDERR --mount src=c:\Users\v-bryanl\code\bdlb77\secrets-store-csi-driver-provider-azure,dst=/go/src/secrets-store-csi-driver-provider-azure,type=bind,consistency=cached --mount source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind --mount source=/.azure,target=/home/vscode/.azure,type=bind -l vsch.quality=stable -l vsch.remote.devPort=0 -l vsch.local.folder=c:\Users\v-bryanl\code\bdlb77\secrets-store-csi-driver-provider-azure --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --entrypoint /bin/sh vsc-secrets-store-csi-driver-provider-azure-8bb2d1eda78a89148ffcaaf21fbc8348 -c echo Container started ; while sleep 1; do :; done
```
Here is the error if I use `${env:HOME}`.. and if `${HOME}` is used.. the path isn't resolved. I updated the time on my machine just to be sure that it wasn't the time causing the issue. Same error